regolith/helpers/l_generalhelper.py

```diff
@@ -14,19 +14,22 @@
 def subparser(subpi):
     subpi.add_argument(
         "coll",
-        help='collection where the lister is going to run.')
+        help='Collection where the lister is going to run.')
     subpi.add_argument(
         "-f","--kv_filter",
         nargs="+",
-        help='search the given collection by key element pairs')
+        help="Search the given collection by key element pairs. "
+             "e.g. < -f name simon >, will return the id of all the people "
```

Comment:
put in the full command, so "e.g., 'regolith helper lister -f name simon' will return...."

Reply:
done!